portfolio/src/main/webapp/script2.js

```diff
@@ -60,28 +65,32 @@ function tabulateAnswers() {
     // if the radio is checked..
     if (choices[i].checked) {
       // add 1 to that choice's score
-      if (choices[i].value =cvals[i]) {
-        cscores[i] = cscores[i] + 1;
-        console.log('here!');
+
+      cdict[choices[i].value] = cdict[choices[i].value] + 1; 
+      
       }
     }
-  }
   // Find out which choice got the highest score.
-  var maxscore = Math.max.apply(Math,cscores);
+  console.log(cdict);
+  var vals = Object.keys(cdict).map(function(key){
+    return cdict[key]});
+  
+  var maxscore = Math.max.apply(Math,vals);
+  console.log('maxscore:')
   console.log(maxscore);
   
   // Display answer corresponding to that choice
   var answerbox = document.getElementById('answer');
-  if (cscores[0] == maxscore) { 
+  if (cdict['c1'] == maxscore) { 
```

Comment:
This looks much better than `cscores[0]` from before.  At least now we know we're checking for c1 specifically, compared to before where you'd need to figure out what was stored in space #0 (zero in this case would be called a 'magic number').

For future improvement, next step here would be to store these in the dict as well (build dict of objects instead of dict of ints) so you wouldn't have to do a chain of IF/ELSE statements.